pylib/anki/errors.py

```diff
@@ -47,7 +47,15 @@ class ExistsError(Exception):
     pass
 
 
-class DeckIsFilteredError(Exception):
+class DeckRenameError(Exception):
+    """Legacy error, use DeckIsFilteredError instead."""
+
+    def __init__(self, description: str, *args: object) -> None:
+        super().__init__(description, *args)
+        self.description = description
+
+
+class DeckIsFilteredError(StringError, DeckRenameError):
```

Comment:
What do you think about this solution? It means we can use `DeckIsFilteredError` just like any other `StringError`, but legacy code can still catch it as `DeckRenameError` and access its `description` field.